### DISCUSSION TITLE
Welcome to readest Discussions!

### DISCUSSION THREAD

**Discussion by chrox** [readest/readest#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FredericDobigny** [comment]
About: FR: In scrolling mode, you can scroll the entire book instead of a chapter #1322
You're right, it's not very bad. It requires some manipulation at the end of the chapter, as it often moves on to the next chapter before I've read the last few sentences. So I have to go back to the previous chapter to finish reading, then go back to the next one.
The PocketBook reader doesn't have this problem, but I admit that your program is better in every way.
I congratulate you on this program because it is the best I have used in 20 years and I read a lot of books on my smartphone.

**Comment by nihaodddd** [comment]
When you read a picture pdf, you can not view the pdf by zooming the picture with two fingers. If only could be as sensitive as looking at a cell phone photo. And at the same time you can't scroll to the next page when you turn on scroll mode
